ui/statistics/src/main/java/de/danoeh/antennapod/ui/statistics/years/BarChartView.java

```diff
@@ -89,17 +89,19 @@ public void draw(@NonNull Canvas canvas) {
             paintBars.setStrokeWidth(height * 0.015f);
             paintBars.setColor(colors[0]);
             int colorIndex = 0;
-            int lastYear = data.size() > 0 ? data.get(0).getYear() : 0;
+            int lastYear = data.size() > 0 ? data.get(data.size() - 1).getYear() : 0;
+            int prevYear = data.size() > 0 ? data.get(0).getYear() : 0;
+            int timeSpan = lastYear - prevYear;
+            boolean firstYearDisplay = timeSpan < 3;
+
             for (int i = 0; i < data.size(); i++) {
                 float x = textPadding + (i + 1) * stepSize;
-                if (lastYear != data.get(i).getYear()) {
-                    lastYear = data.get(i).getYear();
+                if ((prevYear != data.get(i).getYear()) || ((i == 0) && firstYearDisplay)) {
+                    prevYear = data.get(i).getYear();
                     colorIndex++;
                     paintBars.setColor(colors[colorIndex % 2]);
-                    if (i < data.size() - 2) {
-                        canvas.drawText(String.valueOf(data.get(i).getYear()), x + stepSize,
-                                barHeight + (height - barHeight + textSize) / 2, paintGridText);
-                    }
+                    canvas.drawText(String.valueOf(data.get(i).getYear()), x + (stepSize / 3),
+                            barHeight + (height - barHeight + textSize) / 2, paintGridText);
                     canvas.drawLine(x, height, x, barHeight, paintGridText);
```

Comment:
Now with `firstYearDisplay`, it shows an axis tick (usually standing for new year) at a position that is not new year. Please only show the tick if we are actually drawing it for January